plugins/nv-filters/nvidia-videofx-filter.c

```diff
@@ -278,6 +282,12 @@ static bool nvvfx_filter_create_internal(struct nvvfx_data *filter)
 	return true;
 }
 
+static void nvvfx_logger_calback(void *data, const char *msg)
+{
+	UNUSED_PARAMETER(data);
+	blog(LOG_ERROR, "Details - Error reported for NVIDIA VIDEO EFFECTS: %s", msg);
```

Comment:
Logging needs prefix. Additionally, if log levels are inclusive, won't both `FATAL` and `ERROR` logs be grouped under LOG_ERROR?

Reply:
added a prefix. Note that I can't use the error function because data is not a valid pointer when the filter is destroyed. The sdk does not allow the callback to be stopped, oversight from the sdk authors I assume.
I assume FATAL and ERROR will be lumped but haven't checked; this would follow regular convention though.